app/src/main/java/org/akvo/flow/presentation/navigation/SurveyDeleteConfirmationDialog.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2017 Stichting Akvo (Akvo Foundation)
+ *
+ * This file is part of Akvo Flow.
+ *
+ * Akvo Flow is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Akvo Flow is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Akvo Flow.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package org.akvo.flow.presentation.navigation;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v4.app.DialogFragment;
+
+import org.akvo.flow.R;
+
+import static org.akvo.flow.util.ConstantUtil.SURVEY_GROUP_ID_EXTRA;
+
+public class SurveyDeleteConfirmationDialog extends DialogFragment {
+
+    public static final String TAG = "SurveyDeleteConfirmationDialog";
+
+    private int surveyGroupId;
+    private SurveyDeleteListener listener;
+
+    public SurveyDeleteConfirmationDialog() {
+    }
+
+    public static SurveyDeleteConfirmationDialog newInstance(long surveyGroupId) {
+        SurveyDeleteConfirmationDialog fragment = new SurveyDeleteConfirmationDialog();
+        Bundle args = new Bundle();
+        args.putLong(SURVEY_GROUP_ID_EXTRA, surveyGroupId);
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        surveyGroupId = getArguments().getInt(SURVEY_GROUP_ID_EXTRA);
+    }
+
+    @Override
+    public void onAttach(Context context) {
+        super.onAttach(context);
+        try {
+            listener = (SurveyDeleteListener) getActivity();
+        } catch (ClassCastException e) {
+            throw new ClassCastException(getTargetFragment().toString()
+                    + " must implement SurveyDeleteListener");
+        }
+    }
+
+    @Override
+    public void onDetach() {
+        super.onDetach();
+        listener = null;
+    }
+
+    @NonNull
+    @Override
+    public Dialog onCreateDialog(final Bundle savedInstanceState) {
+        super.onCreateDialog(savedInstanceState);
+        AlertDialog.Builder builder = new AlertDialog.Builder(getContext());
+        builder.setMessage(R.string.delete_project_text)
+                .setCancelable(true)
+                .setPositiveButton(R.string.okbutton,
```

Comment:
I am not sure whats the pattern all over the app but on the online space, we are trying to use the `Delete` button to confirm rather than `ok` button.

Reply:
Yes, there is a specific issue for that and it will be handled in future PR https://github.com/akvo/akvo-flow-mobile/pull/836 (Part 8)